app/src/main/java/com/jayd/swissarmyknife/activities/MainActivity.java

```diff
@@ -1,33 +1,94 @@
 package com.jayd.swissarmyknife.activities;
 
 import android.os.Bundle;
+import android.support.design.widget.TabLayout;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.view.ViewPager;
 import android.support.v7.app.AppCompatActivity;
-import android.widget.Button;
-import android.widget.EditText;
+import android.support.v4.app.FragmentPagerAdapter;
+import android.support.v7.widget.Toolbar;
 
-import com.jayd.swissarmyknife.MyApplication;
 import com.jayd.swissarmyknife.R;
-import com.jayd.swissarmyknife.models.Note;
+import com.jayd.swissarmyknife.fragments.CalendarFragment;
+import com.jayd.swissarmyknife.fragments.ExpenseFragment;
+import com.jayd.swissarmyknife.fragments.NoteFragment;
 import com.jayd.swissarmyknife.room.note.NoteDBHelper;
 
-import java.util.UUID;
-
 public class MainActivity extends AppCompatActivity {
 
     private NoteDBHelper noteDBHelper;
+    private Toolbar toolbar;
+    private TabLayout tabLayout;
+    private ViewPager viewPager;
+    private ViewPagerAdapter viewPagerAdapter;
+
+    //Tab numbers
+    private static final int Calendar = 0;
+    private static final int Notes = 1;
+    private static final int Expenses = 2;
```

Comment:
Can we make these public and all capitalized? i.e. public static final int CALENDAR = 0;